recipes/model-bakery/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "model_bakery" %}
+{% set version = "1.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: aea339f47aa3b044d1a6d62f54c04686285fa0397a05de29ec6809a2a6f9bdc5
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
```

Comment:
This looks like it is a candidate for `noarch: python`:

https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python